src/main.js

```diff
@@ -17,39 +21,93 @@ const filmCards = new Array(FILM_COUNT)
   .map((element, idElement) => generateCardFilm(idElement));
 const mainElement = document.querySelector('.main');
 const headerElement = document.querySelector('.header');
+const page = document.querySelector('body');
 
-render(headerElement, createUserProfileTemplate, 'beforeend');
-render(mainElement, createMenuTemplate(filmCards), 'beforeend');
-render(mainElement, createSortCardTemplate, 'beforeend');
-render(mainElement, '<section class="films"></section>', 'beforeend');
+renderElement(headerElement, new UserProfileView().getElement(), RenderPosition.BEFOREEND);
+renderElement(mainElement, new SiteMenuView(filmCards).getElement(), RenderPosition.AFTERBEGIN);
+renderElement(mainElement, new SortCardMenuView().getElement(), RenderPosition.BEFOREEND);
+renderElement(mainElement, new GenericMovieWrapperView().getElement(), RenderPosition.BEFOREEND);
 
 const allMovieWrapper = mainElement.querySelector('.films');
 
+const renderCardFilm = (wrapper, card) => {
+  const filmCard = new FilmCardView(card).getElement();
```

Comment:
аналогично вынести за пределы функции создание инстанса

```suggestion
  const filmCard = filmCardView.getElement();
```